### PR TITLE
remove graphql query for accessDetails as it is return by aquarius

### LINKED ISSUE #1994: Remove subgraph query from algorithm component
In the algorithm component we are doing an aquarius query and then a subgraph query to get the prices. Keep the query when you select the algorithm, we want to calculate the price once the user selects an algorithm
image

In the list we can just put the price we get from aquarius, it's good enough.

### PR DESCRIPTION
**Pull request by EnzoVezzaro**
- Fixes #1994 .

Changes proposed in this PR:

- removed subgraph query for accessDetails during init

### CODE REVIEW

**Comment by EnzoVezzaro** [comment]
@nickscavuzzo33 can you please check if you can find the algo you're looking for on this PR.

@bogdanfazakas can you please check if ti's all good, specially with the changes done to the `accessDetails`

**Comment by nickscavuzzo33** [comment]
Algo Found - @bogdanfazakas @mihaisc should be goodoto merge

File: src/components/Asset/AssetActions/Compute/index.tsx [hunk 1]
```diff
@@ -187,6 +187,8 @@ export default function Compute({
         signer,
         algoProviderFees
       )
+      console.log(algorithmOrderPriceAndFees)
```
Comment (bogdanfazakas) [inline src/components/Asset/AssetActions/Compute/index.tsx:190]: Just remove these logs and should be good to go

File: src/components/Asset/AssetActions/Compute/index.tsx [hunk 2]
```diff
@@ -241,6 +244,8 @@ export default function Compute({
           asset.metadata.type
         )[0]
       )
+      console.log(initializedProvider?.datasets?.[0]?.providerFee)
```
Comment (bogdanfazakas) [inline src/components/Asset/AssetActions/Compute/index.tsx:247]: Same

File: src/components/Asset/AssetActions/Compute/index.tsx [hunk 3]
```diff
@@ -267,6 +272,7 @@ export default function Compute({
   }, [asset?.accessDetails, accountId, isUnsupportedPricing])
 
   useEffect(() => {
+    console.log(selectedAlgorithmAsset)
```
Comment (bogdanfazakas) [inline src/components/Asset/AssetActions/Compute/index.tsx:275]: +1

File: src/@context/Asset.tsx [hunk 4]
```diff
@@ -118,25 +118,6 @@ function AssetProvider({
     [did, accountId]
   )
 
-  // -----------------------------------
-  // Helper: Get and set asset access details
-  // -----------------------------------
-  const fetchAccessDetails = useCallback(async (): Promise<void> => {
-    if (!asset?.chainId || !asset?.services?.length) return
```
Comment (bogdanfazakas) [inline src/@context/Asset.tsx:125]: Should we still remove this since the updated the query size ?

**Review by bogdanfazakas — APPROVED** [review]

**Comment by AnaLoznianu** [comment]
@nickscavuzzo33 If you can validate these changes in production it would be great

**Pull request merged**